doc/api/assert.md

```diff
@@ -1063,6 +1063,31 @@ If the values are not strictly equal, an `AssertionError` is thrown with a
 `message` parameter is an instance of an [`Error`][] then it will be thrown
 instead of the `AssertionError`.
 
+## assert.stringMatching(input, match)
+<!-- YAML
+added: v10.11.1
+-->
+* `input` {string|string[]}
+* `match` {RegExp|RegExp[]|string|string[]}
+
+Verify that requested constraints match the provided input :
+
+```js
+const assert = require('assert');
+
+// default case : check if string(s) match criterias
+// should be false
+assert.stringMatching('blublu', ['blublu', 'blu']);
```

Comment:
Does this mean `'blublu' === 'blublu' && 'blublu' === 'blu'`?

Reply:
In short : yes . It is the basic case of the method. Regex make it more customizable tests